### PR TITLE
chore(glide): add lockfile

### PR DESCRIPTION
**Pull request by technosophos**

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -1,16 +1,12 @@
...
-  version: dd632973f1e7218eb1089048e0798ec9ae7dceb8
 - package: gopkg.in/v2/yaml
-  version: 53feefa2559fb8dfa8d81baad31be332c97d6c77
 - package: gopkg.in/yaml.v2
-  version: 53feefa2559fb8dfa8d81baad31be332c97d6c77
 - package: github.com/deis/pkg/aboutme
-  version: f63d9718b86f17c9320f838fb3eb6039cadb9691
+  version: ">=0.2.1"
 - package: github.com/google/protobuf
-  version: b640f216a56ae25aab9a1fc6f21aa1a56c304494
```
Comment (arschles) [inline glide.yaml:16]: Glide set the versions in the lockfile to different from what was in here. Did you test with the new versions?

Comment (technosophos) [inline glide.yaml:16]: Do you know what the story was for all of these hash pins? I removed so that k8s would pin to its own versions.

Comment (arschles) [inline glide.yaml:16]: Yea, I this was the only version setup that would get Minio building...

_but_ now that glide can produce a lockfile I'm ok with letting it do its thing

Comment (technosophos) [inline glide.yaml:16]: Okay. I'll revert and regenerate the lock.

Comment (arschles) [inline glide.yaml:16]: Sounds good. After alpha, I think it'd be a good idea to remove the pins, regenerate the lock and test everything with that. Feels more glide 0.8-ey to me. Thoughts?

**Pull request closed without merging**